### DISCUSSION TITLE
Plans for the Management Cluster

### DISCUSSION THREAD

**Discussion by PatrickLaabs** [SovereignCloudStack/issues#218] [General]
Hey everyone, 

I am fairly new into the project and still getting my head around it. 
But I am wondering.. are there any plans to support more applications for the management cluster in the first place? 

Like using flux/Argo to automate the deployment and (maybe good first configs) for some base applications like velero, traefik, istio, grafana Loki, fluentd.. you name it. 

This could also be used for any attached cluster, and might help the users to get started in the first place.
But maybe it already exists and I just cannot find it? 😄 

If not, I'd love to get involved 😊 

Best
Patrick

**Comment by garloff** [comment]
Seems we need to improve monitoring the github discussions. Sorry for the late response!
1. The current reference implementation (github/SCS/k8s-cluster-api-provider) does include a one-switch option to opt-in for flux.
  We don't have any of the other services that you mention though.
2. The medium-term plan is actually to focus managed-KaaS operators on rather bare clusters -- rather provide well-standardized clusters (with CCM, CSI, CNI, and provider-mandatory/user-optional Metrics and Ingress) well.
3. The good standardization will then allow a runtime-service catalog to work across all SCS compatible KaaS providers. Yes, no special annotations or similar annoyances.

HTH!

**Comment by garloff** [comment]
And your involvement would be highly appreciated.
Join us in our team meets (Mon 10:05, restarting Jan 9), in our github discussions (start e.g. with epic #181 in this issues repo).
